packages/modules/src/modules/banners/globalNYMoment/GlobalNYMomentBanner.stories.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { getMomentTemplateBanner } from '../momentTemplate/MomentTemplateBanner';
+import { props } from '../utils/storybook';
+import { BannerProps, SecondaryCtaType } from '@sdc/shared/types';
+import { bannerWrapper } from '../common/BannerWrapper';
+import { brand, brandAlt, neutral } from '@guardian/src-foundations';
+
+export default {
+    title: 'Banners/MomentTemplate',
+    args: props,
+} as Meta;
+
+const GlobalNewYearBanner = bannerWrapper(
+    getMomentTemplateBanner({
+        backgroundColour: '#F1F8FC',
+        headerSettings: {
+            textColour: '#0077B6',
+        },
+        primaryCtaSettings: {
+            default: {
+                backgroundColour: '#0077B6',
+                textColour: 'white',
+            },
+            hover: {
+                backgroundColour: '#004E7C',
+                textColour: 'white',
+                border: `1px solid ${'#004E7C'}`,
+            },
+        },
+        secondaryCtaSettings: {
+            default: {
+                backgroundColour: '#F1F8FC',
+                textColour: '#004E7C',
+                border: `1px solid ${'#004E7C'}`,
+            },
+            hover: {
+                backgroundColour: '#E5E5E5',
+                textColour: '#004E7C',
+                border: `1px solid ${'#004E7C'}`,
+            },
+        },
+        closeButtonSettings: {
+            default: {
+                backgroundColour: '#F1F8FC',
+                textColour: brand[400],
+                border: `1px solid ${brand[400]}`,
```

Comment:
One more here in the stories file then just needs to be replicated in the main banner file

Reply:
Could you give me a little more detail here, sorry not sure what I have missed. closeButtonSettings the same in the stories file and used/declared ok in MomentTemplateBanner.tsx/settings.tsx I think. Seems to work ok? Will be removing Z-Order in a bit.

Reply:
Think related to ZIndex which have removed, so have closed this if correct.

Reply:
Yes sorry I did mean the z-index. I'm also going to unresolve the flagged issues so I can keep track with the review but will resolve them again once I've gone through and had a look

Reply:
Ok think this is ok as reinstated ZIndex: 100 in Moment Template banner itself)